src/reducers/systemIntakeReducer.ts

```diff
@@ -149,6 +152,17 @@ function systemIntakeReducer(
           ...prepareSystemIntakeForApp(action.payload)
         }
       };
+    case postIntakeNote.SUCCESS:
+      return {
+        ...state,
+        notes: [
```

Comment:
It was previously suggested to make notes as an object with a key as intake ID, and value as an array of notes.

That's still a good idea, but I think I have an idea that will be simple to implement. That will come when I implement viewing notes.